expo/gbe/menu.py

```diff
@@ -0,0 +1,68 @@
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from cms.menu_bases import CMSAttachMenu
+from menus.base import Menu, NavigationNode
+from menus.menu_pool import menu_pool
+
+from gbe.functions import *
+from gbe.models import *
+
+'''
+  This is the best simulation of our old login menu I could come up with - the Django Menu
+  doesn't offer Clickable top level menu items if there are children items.
```

Comment:
This is a Good Thing. Having the top level menu items clickable is a bug, not a feature.